tests/dummy/app/models/ember-flexberry-dummy-comment.js

```diff
@@ -26,6 +26,11 @@ var Model = BaseModel.extend({
 
   // Model validation rules.
   validations: {
+    author: {
+      presence: {
+        message: 'Заполните поле "Автор"'
```

Comment:
If you already defined translations for validation messages, use them then through i18n instead of such static strings.